tests/behaviour/typeql/steps.rs

```diff
@@ -116,6 +117,7 @@ generic_step_impl! {
     }
 
     #[step(expr = "answer size is: {int}")]
+    #[step(expr = "verify answer size is: {int}")]
```

Comment:
Interesting! I believe we should split the 'reasoner'-specific steps into their own package and source, but each step that can be implemented re-using TypeQL steps can then call these methods.

For example:
```
#[step(expr = "verify answer size is: {int}")]
async fn verify_answer_size_is(context: &mut Context, expected_answers: usize) {
  answer_size(context: &mut Context, expected_answers: usize).await
}
```

This will lead to slightly more code, but also maintain the partitioning of the higher-level domain (BDD has essentially two parts: languages and reasoner). Hope that makes sense :)